core/src/main.cpp

```diff
@@ -866,8 +880,17 @@ auto main(int _argc, char* _argv[]) -> int
 		// JSON configs needs to be in main scope to last the entire duration of the program
 		nlohmann::json oi_config;
 		nlohmann::json endpoint_config;
-		load_oidc_configuration(http_server_config, oi_config, endpoint_config);
 
+		// Check if OIDC config exists, skip setup if missing.
+		if (auto openid_config{http_server_config.find(json::json_pointer{"/authentication/openid_connect"})}; openid_config != http_server_config.end()) {
+			oi_config = *openid_config;
+			if (!load_oidc_configuration(oi_config, endpoint_config)) {
```

Comment:
Is `oi_config` needed?

Reply:
If we're talking about the json object, I believe so. Could be wrong, though.

Reply:
This has been resolved.